stan/math/rev/scal/fun/fma.hpp

```diff
@@ -219,7 +224,8 @@ inline var fma(double a, double b, const var& c) {
  * @param c Summand.
  * @return Product of the multiplicands plus the summand, ($a * $b) + $c.
  */
-inline var fma(double a, const var& b, const var& c) {
+template <typename Ta, typename = enable_if_arithmetic<Ta>>
+inline var fma(Ta a, const var& b, const var& c) {
   return var(new internal::fma_vdv_vari(b.vi_, a, c.vi_));  // a-b symmetry
 }
```

Comment:
Out of the scope of this PR but it would be nice to rewrite these with one `fma_vari` method that has specializations of chain methods for the different implimentations in one class. Then all these specializations could be removed